foreshadow/tests/test_feature_engineerers/test_feature_engineerers.py

```diff
@@ -0,0 +1,77 @@
+"""Test feature_engineerer.py"""
+import pandas as pd
+
+from foreshadow.core.column_sharer import ColumnSharer
+from foreshadow.feature_engineerers import (
+    FeatureEngineerer,
+    SmartFeatureEngineerer,
+)
+
+
+def test_feature_engineerer_fit():
+    """Test basic fit call."""
+    import numpy as np
+
+    data = pd.DataFrame(
+        {
+            "age": [10, 20, 33, 44],
+            "weights": [20, 30, 50, 60],
+            "financials": ["$1.00", "$550.01", "$1234", "$12353.3345"],
+        },
+        columns=["age", "weights", "financials"],
+    )
+    cs = ColumnSharer()
+    cs["domain"]["age"] = "personal"
+    cs["domain"]["weights"] = "personal"
+    cs["domain"]["financials"] = "financial"
+
+    cs["intent"]["age"] = "NumericIntent"
+    cs["intent"]["weights"] = "NumericIntent"
+    cs["intent"]["financials"] = "FinancialIntent"
+
+    dc = FeatureEngineerer(cs)
+    dc.fit(data)
+    data = dc.transform(data)
```

Comment:
Since we're comparing input to output here, if we assign data to different var, we can simply compare against data here instead of redefining the same data.